### PR TITLE
Fix the Tezos’ fees field (actually fee) when getting fees from the external API.

### PR DESCRIPTION
**Pull request by phaazon**
Fixes LLC-589.

### CODE REVIEW

**Comment by phaazon** [comment]
@KhalilBellakrid sure! :)

**Comment by gre** [comment]
just to be clear. this will not be complete in term of fixes needed. see https://ledgerhq.atlassian.net/browse/LLC-591

**Comment by KhalilBellakrid** [comment]
@phaazon Another thing, it seems that even by fixing this, there are some issues on some amounts of txs etc ... We will have to investigate that because it might be related to same issue, what we can do is build a local version with this fix and run `live-common`'s tests and we'll realize the regressions !

File: core/idl/wallet/tezos/tezos_like_wallet.djinni [hunk 1]
```diff
@@ -154,9 +154,10 @@ TezosLikeOriginatedAccount = interface +c {
 }
 
 TezosConfiguration = interface +c {
-  	const TEZOS_XPUB_CURVE: string = "TEZOS_XPUB_CURVE";
+  const TEZOS_XPUB_CURVE: string = "TEZOS_XPUB_CURVE";
 	const TEZOS_PROTOCOL_UPDATE: string = "TEZOS_PROTOCOL_UPDATE";
 	const TEZOS_NODE: string = "TEZOS_NODE";
+  const TZSTATS_API_ENDPOINT: string = "TZSTATS_API_ENDPOINT";
```
Comment (KhalilBellakrid) [inline core/idl/wallet/tezos/tezos_like_wallet.djinni]: Why do we need this ? I think for comparaison you'll have to retrieve: `BLOCKCHAIN_EXPLORER_API_ENDPOINT` and compare it against default `TZSTATS_API_ENDPOINT` 🤔

**Comment by phaazon** [comment]
> just to be clear. this will not be complete in term of fixes needed. see https://ledgerhq.atlassian.net/browse/LLC-591

Yes, that PR is only about the `fees` situation! :)

File: core/idl/wallet/tezos/tezos_like_wallet.djinni (hunk 1 above)
Comment (phaazon) [reply core/idl/wallet/tezos/tezos_like_wallet.djinni]: Oh you are right. Nice catch!

**Pull request merged**